airflow-core/docs/public-airflow-interface.rst

```diff
@@ -25,6 +36,13 @@ and extending Airflow capabilities by writing new executors, plugins, operators
 Public Interface can be useful for building custom tools and integrations with other systems,
 and for automating certain aspects of the Airflow workflow.
 
+In Airflow 3.0+, the primary public interface for DAG authors and task execution is the
+:doc:`airflow.sdk namespace <core-concepts/taskflow>`. Direct access to the metadata database
+from task code is no longer allowed. Instead, use the :doc:`Stable REST API <stable-rest-api-ref>`,
+`Python Client <https://github.com/apache/airflow-client-python>`_, or Task Context methods.
```

Comment:
```suggestion
`Python Client <https://github.com/apache/airflow-client-python>`_, or Task Context methods.
```
`Task Context methods.` is not v clear to me.

Reply:
Changed it